package.json

```diff
@@ -7,20 +7,21 @@
   "bugs": {
     "url": "https://github.com/yankaindustries/mc-components/issues"
   },
-  "main": "./dist/index.js",
-  "module": "./dist/index.js",
-  "style": "dist/styles/css/mc-components.css",
+  "main": "./dist/components/index.js",
```

Comment:
The new structure has everything in either:
- `/dist/components/`
- `/dist/styles/scss/`
- `/dist/styles/css/`